python/cuspatial/cuspatial/core/spatial/indexing.py

```diff
@@ -147,6 +172,15 @@ def quadtree_on_points(
         Length: 120, dtype: int32
     """
 
+    if min_size is not None and max_size is None:
+        max_size = min_size
+        min_size = None
+        warnings.warn(
```

Comment:
https://github.com/rapidsai/cuspatial/issues/798 files to remove this in 23.02 after we deprecate it in 22.12.